packages/cubejs-query-orchestrator/src/orchestrator/QueryCache.js

```diff
@@ -12,9 +13,23 @@ export class QueryCache {
     this.driverFactory = clientFactory;
     this.externalDriverFactory = options.externalDriverFactory;
     this.logger = logger;
-    this.cacheDriver = options.cacheAndQueueDriver === 'redis' ?
-      new RedisCacheDriver({ pool: options.redisPool }) :
-      new LocalCacheDriver();
+
+    console.log('setting up the stuff');
+    console.log('DRIVER:', options.cacheAndQueueDriver);
+
+    switch(options.cacheAndQueueDriver) {
+      case 'redis':
+        this.cacheDriver = new RedisCacheDriver({ pool: options.redisPool });
+        break;
+      case 'dynamodb':
+        this.cacheDriver = new DynamoDBCacheDriver({ tableName: options.tableName });
+        break;
+      case 'memory':
+        this.cacheDriver = new LocalCacheDriver();
+        break;
+      default:
+        throw new Error('Cache Not Specified')
```

Comment:
Should this default to memory? It gets passed in from QueryOrchestrator so it should never actually be empty